renderer/pages/settings.tsx

```diff
@@ -1,84 +1,63 @@
 import { useEffect, useState } from "react";
 import Link from "next/link";
 import { shallow } from "zustand/shallow";
-import FolderSelector from "../components/FolderSelector";
-import { useMainStore } from "../store/mainStore";
 import { useThemeStore } from "../store/themeStore";
-import TrelloConnection from "../components/TrelloConnection";
-import Office365Connection from "../components/Office365Connection";
-import GoogleConnection from "../components/GoogleConnection";
-import clsx from "clsx";
-import TimetrackerWebsiteConnection from "../components/TimetrackerWebsiteConncetion";
-import BetaToggle from "../components/ui/BetaToggle";
-import JiraConnection from "../components/JiraConnection";
 import MenuItem from "../components/ui/MenuItem";
 import ButtonTransparent from "../components/ui/ButtonTransparent";
 import { ChevronLeftIcon } from "@heroicons/react/24/outline";
-
-enum SidebarNavItem {
-  Connections = "Connections",
-  ReportsFolder = "Reports folder",
-  Theme = "Theme",
-  VersionSelect = "Version",
-}
+import { SidebarNavItem, SETTING_SECTIONS } from "../helpers/contstants";
 
 const SettingsPage = () => {
   const [theme] = useThemeStore(
     (state) => [state.theme, state.setTheme],
-    shallow
+    shallow,
   );
   const [isOSDarkTheme, setIsOSDarkTheme] = useState(true);
   const [currentMenuItem, setCurrentMenuItem] = useState<SidebarNavItem>(
-    SidebarNavItem.Connections
+    SidebarNavItem.Connections,
   );
 
+  const settingSections = SETTING_SECTIONS[currentMenuItem];
```

Comment:
I guess this variable should be in singular, like `settingSection` or `currentSettingSection` or `activeSettingSection`

Reply:
Sure

Reply:
Resolved.